internal/formats/common/spdxhelpers/source_info.go

```diff
@@ -29,6 +29,8 @@ func SourceInfo(p pkg.Package) string {
 		answer = "acquired package info from rust cargo manifest"
 	case pkg.PhpComposerPkg:
 		answer = "acquired package info from PHP composer manifest"
+	case pkg.PubPkg:
```

Comment:
nit: maybe this should actually be `DartPubPkg` instead of just `PubPkg`?

Reply:
I'll second this -- `DartPubPkg` better grounds the ecosystem which this belongs without being repetitive.